cli/ceph/rgw.py

```diff
@@ -0,0 +1,24 @@
+from cli import Cli
+from cli.utilities.utils import build_cmd_from_args
+
+
+class Rgw(Cli):
+    """This module provides CLI interface to manage the RGW service."""
+
+    def __init__(self, nodes, base_cmd):
+        super(Rgw, self).__init__(nodes)
+
+        self.base_cmd = f"{base_cmd} rgw"
+
+    def realm(self, action, **kw):
```

Comment:
is this support only `bootstrap` action ? If there are more then better to have separate class for `realm`